### DISCUSSION TITLE
Error: failed to create container: Error response from daemon: No such image: ghcr.io/catthehacker/ubuntu:act-latest

### DISCUSSION THREAD

**Discussion by miohtama** [nektos/act#1264] [Q&A]
I am attempting to run `act` for a private repo on macOS using Docker for Mac.

I am using ` --container-architecture linux/amd64`. 

I am getting the following:

```
act --container-architecture linux/amd64                                                    

[Automated test suite/test] 🚀  Start image=ghcr.io/catthehacker/ubuntu:act-latest
[Automated test suite/test]   🐳  docker pull image=ghcr.io/catthehacker/ubuntu:act-latest platform=linux/amd64 username= forcePull=false
[Automated test suite/test]   🐳  docker create image=ghcr.io/catthehacker/ubuntu:act-latest platform=linux/amd64 entrypoint=["/usr/bin/tail" "-f" "/dev/null"] cmd=[]
[Automated test suite/test] 🏁  Job succeeded
Error: failed to create container: Error response from daemon: No such image: ghcr.io/catthehacker/ubuntu:act-latest
```

I see it is obviously pulling the image, but still somehow fails to find it after the pull.

I already swept the issue tracker and Google for similar errors but did not find anything specific to this error.

Maybe someone could point me to the right track to diagnose this?

**Comment by KnisterPeter** [comment]
Could have been this bug by GitHub: https://github.com/nektos/act/issues/1248
